app/models/match_decisions/record_client_housed_date_shelter_agency.rb

```diff
@@ -77,9 +77,9 @@ def completed
         match.confirm_match_success_dnd_staff_decision.initialize_decision!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.

Reply:
changed.